HACKING.md

```diff
@@ -343,7 +343,8 @@ We have to update the oauth library dependency version:
     sed -Ei 's#<strimzi-oauth.version>[0-9a-zA-Z.-]+</strimzi-oauth.version>#<strimzi-oauth.version>1.0.0-SNAPSHOT</strimzi-oauth.version>#g' \
       pom.xml \
       docker-images/kafka/kafka-thirdparty-libs/2.5.x/pom.xml \
```

Comment:
Should we remove this? 2.5 is not supported in Strimzi main anymore.

Reply:
Ah ... sure
```suggestion
```